src/Diagnostics.ModelsAndUtils/ScriptUtilities/Anonymizer.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Text.RegularExpressions;
+
+namespace Diagnostics.ModelsAndUtils.ScriptUtilities
```

Comment:
Small nitpik comment to put this instead in the Utilities folder versus ScriptUtilities

Reply:
Hey Hawk - I reason I had it under ScriptUtilities is in case some detector wants to use this class to mask PII data from some other strings so I think it makes sense here. Let me know if you think otherwise.